configure.ac

```diff
@@ -1478,6 +1486,29 @@ else
   AC_MSG_NOTICE($libnfs_disabled)
 fi
 
+# libgif
+if test "$use_libgif" != "no"; then
+  AC_CHECK_HEADERS([gif_lib.h],,
+   [if test "$use_libgif" = "yes"; then
+      AC_MSG_ERROR($libgif_not_found)
+      USE_LIBGIF=0
+    elif test "$use_libgif" != "no"; then
+      AC_MSG_NOTICE($libgif_not_found)
+      use_libgif="no"
+      USE_LIBGIF=0
+    fi
+   ])
+  if test "$use_libgif" != "no"; then
+    XB_FIND_SONAME([GIF], [gif], [use_libgif])
```

Comment:
as this is not dlopened at runtime, there is absolutely no reason to require distros having shared libgif.